build.sbt

```diff
@@ -416,7 +448,21 @@ lazy val semanticdbIntegration = project
         s"-P:semanticdb:synthetics:on"
       )
     },
-    javacOptions += "-parameters"
+    javacSemanticdbDirectory := (target.in(Compile).value / "javac-semanticdb"),
+    javaHome in Compile := {
+      // force javac to fork by setting javaHome, otherwise the compiler errors
+      import scala.sys.process._
+      Some(file("/usr/libexec/java_home -v 1.8".lines_!.head))
```

Comment:
This will fail both AppVeyor and Travis CI builds.

Reply:
At the top of my head
```scala
@ sys.props("java.home")
res1: String = "/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home/jre"

@ sys.env("JAVA_HOME")
res2: String = "/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home"
```
system properties are likely more reliable